.github/workflows/full_suite_integration_tests.yml

```diff
@@ -99,7 +99,9 @@ jobs:
 
       - name: Prepare Google Service Account
         env:
+          GOOGLE_CLOUD_PROJECT: flank-open-source
```

Comment:
I know this is a hated solution but for github integration test I want to test if this actually makes a difference.

Reply:
so please add it in `:integration_test build.gradle.kts` using gradle extension `environment(key, value)`

Reply:
@piotradamczyk5 Why do you want it there? This change is temporary and only meant to be for the github actions not for the project. 
I therefore dont see why it must be in gradle?
I honestly am not expecting it to work anyways so will remove it afterwards.

WDYT?

Reply:
Ok, maybe leave a comment to remove it afterwards